.gitignore

```diff
@@ -4,6 +4,8 @@ bin/
 *.created*
 .idea
 .vscode
+vendor/
```

Comment:
Where does this vendor directory come from btw? To my knowledge we shouldn't be producing a vendor directory any more and haven't for some time

Reply:
When developing locally might use `go mod vendor` to manage the dependency. I think it's no harm to keep it?